functions.js

```diff
@@ -12,7 +14,7 @@ function getLetter()
 }
 
 // setup letter grid
-function setupLetters()
+function setupLetters(disableReset)
```

Comment:
Interested in knowing why we are passing a boolean to this function. Above, you call this function with the boolean true, and in another location you call this function without passing an argument. It seems that this function does not need a parameter because we do want to setupLetters every time we click a reset button.